react/features/calendar-sync/components/ConferenceNotification.native.js

```diff
@@ -223,7 +223,10 @@ class ConferenceNotification extends Component<Props, State> {
             const now = Date.now();
 
             for (const event of _eventList) {
-                if (event.url !== _currentConferenceURL) {
+                const eventUrl = getURLWithoutParams(new URL(event.url))
+                    ._url.toLowerCase();
```

Comment:
Without actually testing it myself, this little bit confuses me. getURLWithoutParams returns a URL object? Is the URL object guaranteed to have the ._url property?

Reply:
The event URLs are tested against a pattern in middleware.js:183, so they must have at least a domain and an at least 1 char room

Reply:
Okay, so I found that _url is getting set from the github url-polyfill package. Is it okay for mobile to be accessing it directly? The browser URL most likely does not have _url.

Reply:
And yes I realize the private _url was being accessed before and that I'm just now bringing it up.

Reply:
Yeah, that is not a standard property so I should stop using it - fixed!